### DISCUSSION TITLE
Help! Failed to run Record on Windows 10

### DISCUSSION THREAD

**Discussion by rolandluosz** [llfbandit/record#195] [General]
I ran Record on Windows 10 and after I pressed the record button, it failed to run. The debug message is: flutter: FormatException: Missing extension byte (at offset 1). I don't know why?  I would really appreciate you if you could help me with this. 

D:\flutter\bin\flutter.bat doctor --verbose
[√] Flutter (Channel stable, 3.10.6, on Microsoft Windows [版本 10.0.19045.3208], locale zh-CN)
    • Flutter version 3.10.6 on channel stable at D:\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f468f3366c (5 weeks ago), 2023-07-12 15:19:05 -0700
    • Engine revision cdbeda788a
    • Dart version 3.0.6
    • DevTools version 2.23.1
    • Pub download mirror https://pub.flutter-io.cn

[√] Windows Version (Installed version of Windows is version 10 or higher)

[!] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at C:/Users/welna/AppData/Local/Android/Sdk
    X cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.6.5)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.6.33829.357
    • Windows 10 SDK version 10.0.20348.0

[√] Android Studio (version 2022.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-9586694)

[√] Connected device (4 available)
    • BKL AL00 (mobile) • RKK0218108000887 • android-arm64  • Android 10 (API 29)
    • Windows (desktop) • windows          • windows-x64    • Microsoft Windows [版本 10.0.19045.3208]
    • Chrome (web)      • chrome           • web-javascript • Google Chrome 114.0.5735.199
    • Edge (web)        • edge             • web-javascript • Microsoft Edge 115.0.1901.203

**Comment by llfbandit** [comment]
You can upgrade your project to get the latest windows package `record_windows 1.0.0-beta.2+1`.
Let me know.
